TwitterKit/TwitterKit/TwitterShareExtensionUI/Private/Composer/Attachment Views/TWTRSETweetURLAttachmentView.m

```diff
@@ -132,7 +132,7 @@ - (void)setUpConstraints
     [self.linkDomainLabel.topAnchor constraintEqualToAnchor:self.linkTitleLabel.bottomAnchor].active = YES;
     [self.linkDomainLabel.bottomAnchor constraintEqualToAnchor:self.labelsContainerView.bottomAnchor].active = YES;
 
-    [self.heightAnchor constraintEqualToAnchor:self.linkPreviewImageView.heightAnchor];
+	[self.heightAnchor constraintEqualToAnchor:self.linkPreviewImageView.heightAnchor].active = NO;
```

Comment:
This is the default behavior of a newly created constraint. Quoting from the docs:
> For newly created constraints, the active property is NO by default.

This change is just so that Xcode would not complain with "Ignoring return value of function declared with 'warn_unused_result' attribute"